dev-util/blueprint-compiler/blueprint-compiler-0.18.0.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit edo gnome.org meson python-single-r1 virtualx
+
+DESCRIPTION="Compiler for Blueprint, a markup language for GTK user interfaces"
+HOMEPAGE="https://gnome.pages.gitlab.gnome.org/blueprint-compiler/
+	https://gitlab.gnome.org/GNOME/blueprint-compiler/"
+
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://gitlab.gnome.org/GNOME/${PN}.git"
+else
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
I see gnome.org.eclass does not add any new functions and only sets up SRC_URI and S, so maybe we should put that inside the if-else switch on version? If 9999 - inherit git, else inherit gnome.org.